app/components/search_preview/search_preview.js

```diff
@@ -124,10 +165,10 @@ export default class SearchPreview extends PureComponent {
                 <Animatable.View
                     ref='view'
                     animation='zoomIn'
-                    duration={500}
+                    duration={350}
```

Comment:
Would it make sense to speed this up any more? 350 milliseconds still seems like a pretty long time